Utils/Dataflow/030_PDFAnalyzer/manager.py

```diff
@@ -846,17 +833,16 @@ def export(self, quick=False, outf=False):
         elif quick:
             (text, datasets) = self.find_datasets()
             for category in datasets:
-                d = []
-                for [name, special] in datasets[category]:
-                    d.append(name)
                 outp["content"][category_export_dict[category]
-                                + "_datasets"] = d
+                                + "_datasets"] = [name for
+                                                  [name, special] in
+                                                  datasets[category]]
```

Comment:
Isn\`t it too much for a single line of code?..

Maybe leave temporary variable `d`...
```
            for category in datasets:
                d = [name for [name, special] in datasets[category]]
                outp["content"][category_export_dict[category]
                                + "_datasets"] = d
```
...or create another for `"XXX_datasets"`...
```
            for category in datasets:
                content_type = category_export_dict[category] + "_datasets"
                outp["content"][content_type] = \
                    [name for [name, special] in datasets[category]]
```
...or, if temp variables are unacceptable, this formatting seem to work for `pycodestyle` as well...
```
            for category in datasets:
                outp["content"][category_export_dict[category]
                                + "_datasets"] = \
                    [name for [name, special] in datasets[category]]
```

...or, more globally: define "content" as a separate variable `content = {}` and use this variable instead of `outp["content"]` everywhere:
```
        content = {}
        outp["content"] = content
        ...
            for category in datasets:
                content[category_export_dict[category] + "_datasets"] = \
                    [name for [name, special] in datasets[category]]
```
As both `content` and `outp["content"]` are, actually, just links to same `dict` object, and as `dict` in Python is "mutable" object -- when `content` is changed, `outp` is changed as well:
```
>>> c = {}
>>> outp = {}
>>> outp['c'] = c
>>> c['a'] = 123
>>> print outp
{'c': {'a': 123}}
```